Ginger/Ginger/App.xaml.cs

```diff
@@ -332,8 +341,30 @@ private bool ShouldStartGrid(string[] args)
         /// <returns>ParserResult containing parsed arguments or null.</returns>
         private ParserResult<object> ParseCommandLineArguments(string[] args)
         {
```

Comment:
Update the documentation for the function and also add documentation for each of the condition inside the function

Reply:
done